lib/bulk/ordered.js

```diff
@@ -502,7 +502,7 @@ var executeCommands = function(self, callback) {
     finalOptions.serializeFunctions = true;
   }
 
-  // Serialize functions
+  // Ingore undefined
```

Comment:
`Ignore` not `Ingore`,  see: [Muphry's law](https://en.wikipedia.org/wiki/Muphry%27s_law)